src/components/expanded-state/LiquidityPoolExpandedState.js

```diff
@@ -17,10 +17,10 @@ import {
 import { Chart } from '../value-chart';
 import { ChartPathProvider } from '@rainbow-me/animated-charts';
 import { toChecksumAddress } from '@rainbow-me/handlers/web3';
-import { useChartThrottledPoints } from '@rainbow-me/hooks';
+import { useChartThrottledPoints, useDimensions } from '@rainbow-me/hooks';
 import { magicMemo } from '@rainbow-me/utils';
 
-const heightWithoutChart = 373 + (android ? 20 - getSoftMenuBarHeight() : 0);
+const heightWithoutChart = 220 + (android ? 20 - getSoftMenuBarHeight() : 0);
```

Comment:
this breaks android, may just want to do it conditionally based on OS

Reply:
yes of course!